pkg/loop/plugin_relayer_test.go

```diff
@@ -0,0 +1,484 @@
+package loop_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"math/big"
+	"os"
+	"os/exec"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-plugin"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/libocr/commontypes"
+	"github.com/smartcontractkit/libocr/offchainreporting2/reportingplugin/median"
+	libocr "github.com/smartcontractkit/libocr/offchainreporting2/types"
+
+	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop"
+	"github.com/smartcontractkit/chainlink-relay/pkg/types"
+)
+
+const (
+	account          = libocr.Account("testaccount")
+	balanceCheck     = true
+	blockHeight      = uint64(1337)
+	changedInBlock   = uint64(14)
+	count            = 2
+	epoch            = uint32(88)
+	errMsg           = "test error"
+	from             = "0xabcd"
+	limit            = 42
+	lookbackDuration = time.Minute + 4*time.Second
+	max              = 101
+	n                = 12
+	offset           = 11
+	round            = uint8(74)
+	to               = "0x1234"
+)
+
+var (
```

Comment:
nit: might be nice to have these const + var in a separate file like constants_test or even in helpers_test, just thinking these are used in more than just the plugin relayer tests

(and I struggled to find them earlier haha)

Reply:
`helpers` is not a `package *_test` in this case, but moved consts and vars to `staticdata_test.go` :shrug:

Reply:
`helpers` is not a `package *_test` in this case, but moved consts and vars to `staticdata_test.go` :shrug: